utils.js

```diff
@@ -361,7 +361,8 @@ async function getNotarizationSignatures (notaryAssets, txHex) {
         }
       }
     } else {
```

Comment:
I didn't want to change the way the app handle errors because it might be inconsistent with the rest of the app and changing it would need a wider refractor. The way it is right now is not really standard in JS land, i'd rather throw the error and handle it at top level.